powertools-parameters/powertools-parameters-ssm/src/main/java/software/amazon/lambda/powertools/parameters/ssm/SSMProvider.java

```diff
@@ -124,33 +107,6 @@ public String getValue(String key) {
         return client.getParameter(request).parameter().value();
     }
 
-    /**
-     * {@inheritDoc}
-     */
-    @Override
-    public SSMProvider defaultMaxAge(int maxAge, ChronoUnit unit) {
```

Comment:
I believe this should be mutated on the CacheManager itself, not on this interface. Changing it here breaks encapsulation in a non-intuitive fashion